### PR TITLE
questionnaires for consultations

### PR DESCRIPTION
**Pull request by victazzz**

### CODE REVIEW

File: src/components/Consultations/ConsultationCard/JitsyMeeting/Message/index.tsx [hunk 1]
```diff
@@ -22,7 +41,10 @@ const JitsyMeetingMessage: React.FC<Props> = ({
 }) => {
   const { t } = useTranslation();
   return (
-    <div>
+    <Container className="jitsy-message">
+      <Title size="24" className="jitsy-message__title">
+        Zgoda na analizę emocji i uwagi
```
Comment (mcholcha) [inline src/components/Consultations/ConsultationCard/JitsyMeeting/Message/index.tsx:46]: translation

File: src/components/Consultations/ConsultationCard/JitsyMeeting/index.tsx [hunk 2]
```diff
@@ -159,17 +170,8 @@ const JitsyMeeting: React.FC<Props> = ({
...
-            message={t("ConsultationPage.AdditionalRecording")}
-            closeToast={toast.dismiss}
-            userConsentedRef={userConsentedRef}
-          />,
-          {
-            position: "top-center",
-            autoClose: false,
-          }
-        );
+        console.log("Show modal");
```
Comment (mcholcha) [inline src/components/Consultations/ConsultationCard/JitsyMeeting/index.tsx:173]: log

File: src/components/Consultations/ConsultationTutorCard/Content/DateInfo/index.tsx [hunk 3]
```diff
@@ -25,7 +25,7 @@ const ConsultationTutorCardContentDateInfo = ({ consultation }: Props) => {
   const isApproved = consultation.status === "approved";
   const isNotReported = consultation.status === "not_reported";
   const isRejected = consultation.status === "reject";
-
+  console.log("ConsultationTutorCardContentDateInfo", consultation);
```
Comment (mcholcha) [inline src/components/Consultations/ConsultationTutorCard/Content/DateInfo/index.tsx:28]: log

File: src/components/Courses/Course/CoursePanelLayout/FinishPage/Rate/index.tsx [hunk 4]
```diff
@@ -1,89 +1,245 @@
...
   onFinish,
   onSuccesGetQuestionnaires,
 }: Props) => {
-  const { fetchQuestionnaires, fetchQuestionnaire } =
-    useContext(EscolaLMSContext);
-  const [state, setState] = useState({
+  const {
+    questionnaires: questionnairesList,
+    loading,
+    // error,
```
Comment (mcholcha) [inline src/components/Courses/Course/CoursePanelLayout/FinishPage/Rate/index.tsx:27]: comment

**Review by mcholcha — CHANGES_REQUESTED** [review]

File: src/components/Consultations/ConsultationCard/JitsyMeeting/index.tsx (hunk 2 above)
Comment (KrystianEscolaSoft) [inline src/components/Consultations/ConsultationCard/JitsyMeeting/index.tsx:173]: console.log

File: src/components/Consultations/ConsultationTutorCard/Content/DateInfo/index.tsx (hunk 3 above)
Comment (KrystianEscolaSoft) [inline src/components/Consultations/ConsultationTutorCard/Content/DateInfo/index.tsx:28]: .

**Review by mcholcha — APPROVED** [review]

**Pull request merged**